assets/js/app/common.js

```diff
@@ -2,135 +2,135 @@ import $ from 'jquery';
 import { DateTime } from 'luxon';
 
 $(document).ready(function() {
-  // add a js class to indicate we have JS enabled. Might need a change to either modernizr or somethng comparable
-  $('html').addClass('js');
```

Comment:
This is just space shift due to `.editorconfig` update. 
#til It's nice that npm style lint adapts to it :)